FrontEnd/src/router/index.js

```diff
@@ -62,34 +62,19 @@ const router = createRouter({
       component: () => import("../views/EditAanganwadi.vue")
     },
     {
-      path: "/addSupplement",
```

Comment:
you have delete all these changes which are required while rebasing. Can you bring back all. You should rebase carefully. You can take my or someone's help